src/ui/views/Dashboard/components/BalanceView.tsx

```diff
@@ -105,14 +110,33 @@ const BalanceView = ({
             startAnimate ? numberAnimation : 'text-32'
           )}
         >
-          <div className={clsx('amount-number', !startAnimate && 'text-32')}>
-            <span
-              onClick={onClick}
-              className=" cursor-pointer"
-              title={splitNumberByStep((balance || 0).toFixed(2))}
-            >
-              ${splitNumberByStep((balance || 0).toFixed(2))}
-            </span>
+          <div
+            className={clsx(
+              'amount-number balance-loading',
+              !startAnimate && 'text-32'
+            )}
+          >
+            {(balanceLoading && !balanceFromCache) || balance === null ? (
```

Comment:
```suggestion
            {(balanceLoading && !balanceFromCache) || !balance ? (
```
包括 balance 为 null 和 0 的情况